src/test/java/org/hydev/mcpm/utils/ZstdUtilsTest.java

```diff
@@ -0,0 +1,60 @@
+package org.hydev.mcpm.utils;
+
+import com.google.common.base.Stopwatch;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.concurrent.TimeUnit;
+
+import static org.hydev.mcpm.utils.ZstdUtils.*;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+
+/**
+ * Tests for ZstdUtils
+ *
+ * @author Azalea (https://github.com/hykilpikonna)
+ * @since 2022-11-22
+ */
+class ZstdUtilsTest
+{
+    @Test
+    void test() throws ZstdException
+    {
+        assumeTrue(ZstdUtils.nativeSupport());
+        assertEquals(new String(decompress(compress("Hello world".getBytes()))), "Hello world");
```

Comment:
Worth potentially specifying encoding here for safety (`"UTF-8"`). It's a test so I guess if something goes wrong with encoding we'll know though 🙃

Reply:
Done